js/modules/boost.src.js

```diff
@@ -1884,7 +1884,10 @@ function GLRenderer(postRenderCallback) {
 			);
 		}
 
-		if (!lastX) {
+		if (!lastX &&
+			connectNulls !== false &&
+			closestLeft > Number.MIN_VALUE &&
```

Comment:
`Number.MIN_VALUE` is close to 0. Do we mean `-Number.MAX_VALUE`?

Reply:
`MIN_VALUE` was intentional, but come to think of it, `-Number.MAX_VALUE` is better as it allows for negative x-values. I'll push a fix.